test/e2e/Makefile

```diff
@@ -63,10 +63,10 @@ run: $(TOOLING_BINARIES) $(E2E_DATA_DIR) push-ccm-image
 		--e2e.chart-folder="$(E2E_CHART)" --e2e.skip-resource-cleanup=false \
 		--e2e.version="$(VERSION)"
 
-dev: $(TOOLING_BINARIES) $(E2E_DATA_DIR) push-ccm-image
+dev: $(TOOLING_BINARIES) $(E2E_DATA_DIR)
 	$(GINKGO) -v . -- --e2e.config="$(E2E_CONF_FILE_DEV)" --e2e.artifacts-folder="$(E2E_ARTIFACTS)" \
 		--e2e.chart-folder="$(E2E_CHART)" --e2e.skip-resource-cleanup=true \
-		--e2e.version="$(VERSION)"
+		--e2e.version="v1.22.3-79-gde8bafb"
```

Comment:
Why is this hard coded?

Reply:
this is checked-in by mistake, will revert it